CHANGELOG.rst

```diff
@@ -1,5 +1,38 @@
 Changelog
 =========
+1.7.0 (2020-03-01)
+------------------
+MLflow 1.7.0 includes several major features and improvements:
+
+Model Registry APIs: breaking changes
+- REST APIs are no longer in beta and intended to be stable until the next major version. As such, this release includes a number of breaking changes.
+- Model Registry UI has been updated to use new REST APIs (#2456, #2476 @aarondav; #2507, @mparkhe)
+- MLflow Client APIs are backward compatible and internally been updated to use new REST APIs (#2457, @tomasatdatabricks; #2502, @mparkhe)
+- Java Client API to download models artifacts (#2308, @andychow-db)
+- Updated Model Registry documentation page with code snippets and examples (#2493, dmatrix)
+
+Other Features:
+
+- Fix bar chart bug introduced in #2393 (#2498, @smurching)
```

Comment:
People don't know what #2393 is so say which MLflow release this was in (or even better, which bug)